### ISSUE TITLE
Make Code Panel wider

### ISSUE THREAD

**Issue by flatcap** [fabiocchetti/wide-github#19]
Please can you make the Code Panel full-width too.
Using Firefox's Inspector, the `<div>` that needs altering is:

```xml
<div data-selector="repos-split-pane-content" tabindex="0" class="Box-sc-g0xbh4-0 jgEDtJ">
```

It has the property:

```css
max-width: 1012px;
```

Turning that off improves the look:

### Before

![image](https://github.com/fabiocchetti/wide-github/assets/76760/3b1c09cd-37c3-406e-ba43-9ac57a34e902)

### After

![image](https://github.com/fabiocchetti/wide-github/assets/76760/8ef4dddd-bd55-421d-8be7-1271c48fd624)

That probably means that we need, either/or:

```css
width: 100% !important;
max-width: none !important;
```

Thanks :heart:

**Comment by fabiocchetti** [comment]
Implemented in [version 2.2.0](https://github.com/fabiocchetti/wide-github/releases/tag/2.2.0).

Thanks for reporting!

**Comment by flatcap** [comment]
That looks much better.  Thanks!
